code/modules/mob/living/simple_animal/animals/giant_spider.dm

```diff
@@ -8,7 +8,7 @@
 /mob/living/simple_animal/hostile/giant_spider
 	name = "giant spider"
 	desc = "Furry and brown, it makes you shudder to look at it. This one has deep red eyes."
-	tt_desc = "Atrax robustus gigantus"
+	tt_desc = "X Brachypelma phorus"
```

Comment:
Fun police 🚓 Removing a great pun that's scientifically accurate!

Reply:
wow holy shit i do not like funnel web spiders!